storage-transcribe-audio/functions/src/transcribe-audio.ts

```diff
@@ -61,14 +62,19 @@ export async function transcribeAndUpload({
   }
 
   logs.receivedLongRunningRecognizeResponse(response);
-  const taggedTranscription = response.results?.map(
-    (result) =>
-      [result?.channelTag, result?.alternatives?.[0].transcript] as const
-  );
+  if (response.results == null) {
+    return {
+      status: Status.FAILURE,
+      warnings,
+      type: FailureType.NULL_TRANSCRIPTION,
```

Comment:
Is "null transcription" indicative of just any empty transcription? Like if I passed in an audio file that's just silent, what would be the output? Would it fail as is?

Reply:
If the transcription is an empty string (as it would be if someone passed a silent audio file), a warning is emitted, but there's no error. A null transcription happens if the response doesn't have results, or if one of the results or channels is null, which I think would only be caused by a bug in the API (but still seems, well, possible).